website/static/js/filepage/revisions.js

```diff
@@ -86,6 +86,102 @@ var FileRevisionsTable = {
             });
         };
 
+        // Start of Subscriptions
+        self.subscription = m.prop('adopt_parent');
```

Comment:
This is an incremental step toward being able to subscribe to individual files. I can certainly take this out.